agents/sbd/fence_sbd.py

```diff
@@ -15,6 +17,7 @@
 PATH_NOT_EXISTS = -1
 PATH_NOT_BLOCK = -2
 SBD_PID_FILE = "@SBDPID_PATH@"
+SBD_SYSCONFIG_FILE = "/etc/sysconfig/sbd"
```

Comment:
Use @INITCONFDIR@ instead of /etc/sysconfig here:
https://github.com/ClusterLabs/fence-agents/pull/622